connectors/Dockerfile

```diff
@@ -1,5 +1,12 @@
 FROM node:18.15.0 as connectors
 
+
+RUN  apt-get update && apt-get \
```

Comment:
tested using : 
```
docker run -v $(pwd):/code  --rm -it arico-connectors   /bin/bash     
$>cd /app
$>npx tsx test.ts
```
It worked 👍

Reply:
Would be worth deploying to connectors-edge first just to be sure